charts/opentelemetry-collector/README.md

```diff
@@ -19,7 +19,7 @@ helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm
 To install the chart with the release name my-opentelemetry-collector, run the following command:
 
 ```console
-helm install my-opentelemetry-collector open-telemetry/opentelemetry-collector --set mode=<value>
+helm install my-opentelemetry-collector open-telemetry/opentelemetry-collector --set mode=<value> --set image.repository= "otel/opentelemetry-collector-k8s" --set command.name="otelcol-k8s"
```

Comment:
Oh, we have a different name for the command. I've been always finding this annoying. I wonder if we should just set `otelcol` for all the images... What you think? If you think it makes sense, we can create an issue in the collector repo and bring it to the SIG for discussion. We can do it gradually and support both old and new names for some time.

Reply:
Yes definitely.  I don't see any value to having a unique command in each distro

Reply:
Submitted https://github.com/open-telemetry/opentelemetry-collector-releases/issues/528